fish_prompt.fish

```diff
@@ -367,6 +369,25 @@ function __bobthefish_prompt_dir -d 'Display a shortened form of the current dir
   __bobthefish_path_segment "$PWD"
 end
 
+function __bobthefish_prompt_vi -d 'Display vi mode'
+  if [ "$theme_display_vi" = 'yes' ]
```

Comment:
to be a bit more consistent with the other segments, let's do an early return rather than an if block that wraps the whole function.

```fish
[ "$theme_display_vi" != 'yes' ]; and return
```